src/App.scss

```diff
@@ -1,16 +1,25 @@
+@import "./components/styles/light-dark-theme";
+body {
+  background-color: var(--bgThemeColor1);
+  color: var(--textThemeColor1);
+  text-align: center;
+  transition: background-color 0.5s ease;
+}
+a {
+  color: var(--textThemeColor1);
```

Comment:
try to use scss variables

like:

`$textThemeColor1`

Reply:
We decided to leave css variables in this place